lib/mcb/commands/providers/geocode.rb

```diff
@@ -0,0 +1,23 @@
+name "geocode"
+summary "Batch geocode Providers"
+usage "geocode [options] <id>"
+option :s, "sleep", "time to sleep between each request", argument: :optional, default: 0.25, transform: method(:Float)
+option :b, "batch_size", "batch size", argument: :optional, default: 100, transform: method(:Integer)
+
+instance_eval(&MCB.remote_connect_options)
+
+require "geocoder"
+
+run do |opts, args, _cmd|
+  MCB.init_rails(opts)
+
+  if args.any?
+    Provider.where(id: args.to_a).find_each(batch_size: opts[:batch_size]) do |provider|
```

Comment:
Should this be `provider_code`? I'd think that that's more convenient probably. Also, it'd be smart to use `recruitment_cycle` here. Have a look at how [the `provider list` commands does this](https://github.com/DFE-Digital/manage-courses-backend/blob/8adac1ac5fad452ed91f59a9120702042a73d722/lib/mcb/commands/providers/list.rb#L9), which relies on a standard options being defined in [the `providers` command](https://github.com/DFE-Digital/manage-courses-backend/blob/8adac1ac5fad452ed91f59a9120702042a73d722/lib/mcb/commands/providers.rb#L5).

Reply:
good call

Reply:
@misaka - yep use of provider code make sense but I'm not sure I understand the rationale/need for using `recruitment_cycle` in this instance? i.e. under what circumstances would I want geocode Providers in one particular recruitment cycle but not the others?